### PR TITLE
Update website after 4.11.1 release:

### PR DESCRIPTION
**Pull request by eolivelli**
- fix BP page
- fix versions list
- fix download page and releases history
- add script to publish the website manually
- tagged as "stable" 4.10.0
- tagged as EOL every release < 4.9.2
- removed stale information about next releases (if we do not have a real plan it is better to not write stale information)
- merge BK 4.11.1 release notes

the procedure to publish the new website manually is:
- git clone https://github.com/apache/bookkeeper
- cd bookkeeper/site
- docker/ci.sh
- scripts/publish-website-manually.sh

### CODE REVIEW

**Comment by eolivelli** [comment]
I have already published this version of the website, because of the release 4.11.1

**Review by nicoloboschi — APPROVED** [review]
+1

File: site/_config.yml [hunk 1]
```diff
@@ -25,8 +27,6 @@ versions:
 - "4.6.1"
 - "4.6.0"
 - "4.5.1"
-- "4.5.0"
```
Comment (dmercuriali) [inline site/_config.yml:28]: I think this should be kept

Comment (eolivelli) [reply site/_config.yml:28]: oh you are right  !
good catch

Comment (eolivelli) [reply site/_config.yml:28]: done

**Review by dmercuriali — APPROVED** [review]

**Comment by eolivelli** [comment]
@rdhabalia and @sijie already approved #2418 -> BK 4.11.1 release notes

File: site/community/bookkeeper_proposals.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ using Google Doc.
 
 This section lists all the _bookkeeper proposals_ made to BookKeeper.
 
-*Next Proposal Number: 41*
+*Next Proposal Number: 42*
```
Comment (Ghatage) [inline site/community/bookkeeper_proposals.md:88]: @eolivelli Since 42 is already accepted, shouldn't this be 43?

**Review by Ghatage — APPROVED** [review]
LGTM, minor comment on next BP version.

File: site/community/bookkeeper_proposals.md (hunk 2 above)
Comment (eolivelli) [reply site/community/bookkeeper_proposals.md]: you are right.
I have fixed it

**Comment by eolivelli** [comment]
This is a website only fix, you can ignore CI

**Comment by Ghatage** [comment]
LGTM, I'll merge this in a few hours if there are no more comments on this.

**Pull request merged**